library/oidc.c

```diff
@@ -507,6 +507,22 @@ static int set_blocking(uv_os_sock_t sock) {
     return 0;
 }
 
+void url_decode(const char *src, const char *end, char *dest, size_t dest_size) {
```

Comment:
might as well make this static if we don't want external callers using it.

Reply:
done